src/libs/ReportUtils.ts

```diff
@@ -1172,10 +1172,25 @@ function isJoinRequestInAdminRoom(report: OnyxEntry<Report>): boolean {
     return ReportActionsUtils.isActionableJoinRequestPending(report.reportID);
 }
 
+/**
+ * Checks if report is in read-only mode.
+ */
+function isReadOnly(report: OnyxEntry<Report>): boolean {
```

Comment:
Given the implementation and possible permissions, this name feels kind of wrong. Something like `canWriteInReport` sounds more accurate to me.

Reply:
Okay.

Reply:
Done - 2f17417, ae9af7f.

Reply:
https://github.com/Expensify/App/assets/57314631/548b1281-f800-4bcf-8723-67eba066a30d